### ISSUE TITLE
Support for Plex Login with 2FA

### ISSUE THREAD

**Issue by shrippen** [Taiko2k/Tauon#1083]
Hi,

I am having trouble login into my Plex Server with activated 2FA on my Plex Account. Is there any way to do this without disabling 2FA?

**Comment by Taiko2k** [comment]
Yeah its not currently possible in Tauon. I can add this to the todo list but I can't give a timeframe on that.

**Comment by zquestz** [comment]
Any progress on this?

**Comment by rollecode** [comment]
Was looking for this as well. Some clients do this just by instructing user to add 2FA number right after the password (in the same field).

**Comment by lifeismeta** [comment]
Any update or progress on this? I've tried fiddling with 2fa in username or password but no success so assume it's not implemented at all.
